it/python/requirements-dev.txt

```diff
@@ -98,7 +98,7 @@ httpx[http2]==0.25.0
 
 hyperframe==6.0.1 ; python_full_version >= '3.6.1'
 
-microsoft-kiota-abstractions==0.9.0
+microsoft-kiota-abstractions==0.8.7
```

Comment:
please revert this, I believe #3524 is going to fix this issue. (you could rebase onto that branch)

Reply:
rebased